metadata-ingestion/tests/integration/oracle/common.py

```diff
@@ -0,0 +1,125 @@
+import pathlib
+from typing import Any, Optional
+
+import pytest
+from sqlalchemy.sql.elements import TextClause
+
+from datahub.ingestion.run.pipeline import Pipeline
+from datahub.ingestion.source.sql.oracle import OracleConfig
+from tests.test_helpers import mce_helpers
+
+
+class OracleSourceMockDataBase:
```

Comment:
was there a reason you made this two separate files?

Reply:
To keep common in separate file, like this class can be used by other integration test-cases in future and to keep python file size in readable size